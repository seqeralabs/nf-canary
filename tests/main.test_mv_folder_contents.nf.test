nextflow_process {

    name "Test Process TEST_MV_FOLDER_CONTENTS"
    script "main.nf"
    process "TEST_MV_FOLDER_CONTENTS"

    test("Should run without failures") {

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
