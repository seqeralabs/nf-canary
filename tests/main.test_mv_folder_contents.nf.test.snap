{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    [
                        "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                    ]
                ],
                "1": [
                    "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                ],
                "outfile": [
                    "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                ],
                "outfolder": [
                    [
                        "test.txt:md5,d41d8cd98f00b204e9800998ecf8427e"
                    ]
                ]
            }
        ],
        "timestamp": "2023-08-18T10:33:34+0000"
    }
}